.github/workflows/run-tests.yml

```
@@ -7,18 +7,18 @@ on:
   push:
     branches:
       - master
+      - develop
+      - 'feature**'  # This will match any branch starting with "feature"
```

Maybe this to better follow gitflow naming?
```suggestion
      - 'feature/**'  # This will match any branch starting with "feature"
```